assets/app/data/crowbar/services/crowbar.factory.js

```diff
@@ -41,7 +41,7 @@
 
```

Comment:
I don't think so.
Repository checks for the admin server now is part of the upgrade API entity, not crowbar.
Therefore, in order to keep consistency, I think we should reflect that by moving the `getRepositoriesChecks` method from crowbar.factory.js into upgrade.factory.js